hatch/src/assets/builder.rs

```diff
@@ -119,4 +135,28 @@ impl Builder {
 
     ProjectAsset::new(project_path.to_path_buf(), Windows::name(), contents)
   }
+
+  pub fn catch_header(&self, project: &Project) -> HatchResult<ProjectAsset> {
+    let test_src_path = project.path().join("test/src");
+    let file_name = CatchHeader::name();
+    if !test_src_path.join(file_name).exists() {
+      let catch_header_url = "https://github.com/catchorg/Catch2/releases/download/v2.1.1/catch.hpp";
+      let mut resp = reqwest::get(catch_header_url).with_context(|e| {
+        format!("failed to retrieve catch.hpp : {}", e)
+      })?;
+      let mut content = String::new();
+      resp.read_to_string(&mut content);
```

Comment:
I think there is a .text() method on the Response object. You may be able to shorten this to
```
let resp = reqwest::get(catch_header_url).with_context(|e| {
  format!("failed to retrieve catch.hpp : {}", e)
})?;
let content = resp.text()?;
```